__tests__/app.test.js

```diff
@@ -813,4 +813,27 @@ describe('GET /api/articles (queries)', () => {
             });
         });
     });
+});
+
+describe('GET /api/articles/:article_id (comment_count)', () => {
+    test('200: should have article with the correct object layout', () => {
+        const objectLayout = {
+            article_id: expect.any(Number),
+            title: expect.any(String),
+            topic: expect.any(String),
+            author: expect.any(String),
+            body: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            article_img_url: expect.any(String),
+            comment_count: expect.any(Number),
+        };
+
+        return request(app)
+        .get('/api/articles/1')
+        .expect(200)
+        .then(({ body }) => {
+            expect(body.article).toMatchObject(objectLayout);
```

Comment:
This is a really interesting way of doing it - the way I would suggest would be to expect(body.article).toHaveProperty(comment_count) or something to that effect -really just to be specific in your test suite, but I'm happy for you to merge with this test :)